src/main/io/serial_cli.c

```diff
@@ -358,8 +363,8 @@ static const char * const lookupTableGPSSBASMode[] = {
 };
 #endif
 
-static const char * const lookupTableCurrentSensor[] = {
-    "NONE", "ADC", "VIRTUAL"
+static const char * const lookupTableCurrentMeter[] = {
```

Comment:
Virtual should be probably on index 0 - it makes more sense to have multiple ADC current sensors, not sure about virtual ones